local-cli/runWindows/utils/WindowsStoreAppUtils.ps1

```diff
@@ -150,9 +150,12 @@ function Install-AppFromAppx {
 function Start-Locally {
     param(
         [Parameter(Mandatory=$true, Position=0, ValueFromPipelineByPropertyName=$true)]
-        [string] $ID <# package.appxmanifest//Identity@name #>
-    )
+        [string] $ID, <# package.appxmanifest//Identity@name #>
 
+        [Parameter(Mandatory=$true, Position=1, ValueFromPipelineByPropertyName=$true)]
+        [string] $proxy
```

Comment:
Yep.

---
In reply to: [68452196](https://github.com/ReactWindows/react-native-windows/pull/499#discussion_r68452196) [](ancestors = 68452196)